src/themes/light/theme.css

```diff
@@ -153,171 +153,171 @@ html {
 .nowPlayingBarSecondaryText,
 .programSecondaryTitle,
 .secondaryText {
-    color: #888
+    color: #888;
 }
 
 .actionsheetDivider {
     background: #ddd;
-    background: rgba(0, 0, 0, .14)
+    background: rgba(0, 0, 0, 0.14);
 }
 
 .cardFooter-vibrant .cardText-secondary {
     color: inherit;
-    opacity: .5
+    opacity: 0.5;
 }
 
 .formDialogHeader a,
 .toast {
-    color: #fff
+    color: #fff;
 }
 
 .actionSheetMenuItem:hover {
-    background-color: #ddd
+    background-color: #ddd;
 }
 
 .toast {
     background: #303030;
-    color: rgba(255, 255, 255, .87)
+    color: rgba(255, 255, 255, 0.87);
 }
 
 .appfooter {
     background: #282828;
     color: #ccc;
-    color: rgba(255, 255, 255, .78)
+    color: rgba(255, 255, 255, 0.78);
 }
 
 .nowPlayingBarSecondaryText {
-    color: #999
+    color: #999;
 }
 
 .itemSelectionPanel {
-    border: 1px solid #00a4dc
+    border: 1px solid #00a4dc;
 }
 
 .selectionCommandsPanel {
     background: #00a4dc;
-    color: #fff
+    color: #fff;
 }
 
 .upNextDialog-countdownText {
-    color: #00a4dc
+    color: #00a4dc;
 }
 
 .alphaPickerButton {
     color: #555;
-    color: rgba(0, 0, 0, .7);
-    background-color: transparent
+    color: rgba(0, 0, 0, 0.7);
+    background-color: transparent;
 }
 
 .alphaPickerButton-selected,
 .alphaPickerButton-tv:focus {
     background-color: #00a4dc;
-    color: #fff !important
+    color: #fff !important;
 }
 
 .detailTableBodyRow-shaded:nth-child(even) {
-    background: #f8f8f8
+    background: #f8f8f8;
 }
 
 .listItem-border {
-    border-color: #f0f0f0 !important
+    border-color: #f0f0f0 !important;
 }
 
 .listItem:focus {
-    background: #ddd
+    background: #ddd;
 }
 
 .progressring-spiner {
-    border-color: #00a4dc
+    border-color: #00a4dc;
 }
 
 .mediaInfoText {
     color: #333;
-    background: #fff
+    background: #fff;
 }
 
 .mediaInfoTimerIcon,
 .starIcon {
-    color: #CB272A
+    color: #cb272a;
 }
 
 .emby-input,
 .emby-textarea {
     color: inherit;
     background: #fff;
-    border: .07em solid rgba(0, 0, 0, .158);
-    border-radius: .15em
+    border: 0.07em solid rgba(0, 0, 0, 0.158);
+    border-radius: 0.15em;
 }
 
 .emby-input:focus,
 .emby-textarea:focus {
-    border-color: #00a4dc
+    border-color: #00a4dc;
 }
 
 .emby-select-withcolor {
     color: inherit;
     background: #fff;
-    border: .07em solid rgba(0, 0, 0, .158)
+    border: 0.07em solid rgba(0, 0, 0, 0.158);
 }
 
-.emby-checkbox:checked+span+.checkboxOutline,
+.emby-checkbox:checked + span + .checkboxOutline,
 .emby-select-withcolor:focus {
-    border-color: #00a4dc
+    border-color: #00a4dc;
 }
 
-.emby-checkbox:focus+span+.checkboxOutline {
+.emby-checkbox:focus + span + .checkboxOutline {
     border-color: #000;
 }
 
-.emby-checkbox:focus:not(:checked)+span+.checkboxOutline {
+.emby-checkbox:checked + span + .checkboxOutline,
+.itemProgressBarForeground {
+    background-color: #00a4dc;
```

Comment:
why `color: #fff;` is gone?

Reply:
Git is getting confused. I'm moving stuff around the file. The color #fff is further down.

Reply:
Maybe it's me who's getting confused, but I think that `color; #fff` which is down refers to other element selector, not this one, could you please re-check?

Reply:
No `color: #fff;` in this spot in master: https://github.com/jellyfin/jellyfin-web/blob/master/src/themes/light/theme.css#L267